ocs_ci/helpers/helpers.py

```diff
@@ -3603,3 +3604,225 @@ def get_secret_names(namespace=defaults.ROOK_CLUSTER_NAMESPACE, resource_name=""
     secret_obj = ocp.OCP(kind=constants.SECRET, namespace=namespace)
     secrets_objs = secret_obj.get(resource_name=resource_name)
     return [secret_obj["metadata"]["name"] for secret_obj in secrets_objs["items"]]
+
+
+def mon_quorom_lost():
```

Comment:
You may consider renaming to to create_mon_quorum_loss() or induce_mon_quorum_loss()

Reply:
changed to induce_mon_quorum_loss()